### ISSUE TITLE
Book 05: Sentence 102

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#27]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/0690fdbe-1b0c-4afa-b3dc-f41a557e57f1)

```
# sent_id = Book_05_-_The_Omens.txt:102
# text = Qujtoaia, ca in tezcatlipuca: mjiecpa qujmonaoaltiaia in Coiutl: auh injc coiutl, mjiecpa teiaiacatzacujlia in vtlica.
# text[norm] = Quihtoaya, ca in Tezcatlipoca: miecpa quimonahualtiaya in Coyotl: auh inic coyotl, miecpa teyayacatzacuilia in ohtlica.
# text[orig] = Quj·toaia·,·ca·intezcatlipuca·:·mjiecpa·qujmonaoaltiaia·in·Co¶iutl·:·auh·injc·coiutl·,·mj·iecpa¶teiaiacatzacujlia·in·vtlica·.
# text[osp] = Asi mesmo dezian: que Tezcatlipuca, muchas vezes se trasformaua en vn animal: que llaman coiutl: que es como lobo, y ansi trasformado, ponia se delante de los caminantes, como atajando los el camino: paraque no pasasen adelante:
# text[spa] = Decían que Tezcatlipoca, muchas veces se transformaba en coyote, y de coyote, muchas veces se encontró con la gente en el camino.
# references = A&D/180:7
1	Qujtoaia	ihtoa	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin	0	root	_	Orig=Quj·toaia|Folio=14r,14r|Paragraph=3,3|Line=1,1|Norm=Quihtoaya|Analysed=Yes
2	,	,	PUNCT	_		8	punct	_	Folio=14r|Paragraph=3|Line=1|Norm=,|Analysed=Yes
3	ca	cah|ca	PART	_	_	8	_	_	Folio=14r|Paragraph=3|Line=1|Norm=ca|AmbigNorm=True|Override=True
4	in	in	DET|SCONJ	_	_	5	_	_	Orig=intezcatlipuca|Folio=14r|Paragraph=3|Line=1|Norm=in
5	tezcatlipuca	Tezcatlipoca	PROPN	_		8	_	_	Orig=intezcatlipuca|Folio=14r|Paragraph=3|Line=1|Norm=Tezcatlipoca|Analysed=Yes
6	:	:	PUNCT	_		8	punct	_	Folio=14r|Paragraph=3|Line=1|Norm=:|Analysed=Yes
7	mjiecpa	miecpa	ADV	_		8	_	_	Folio=14r|Paragraph=3|Line=2|Norm=miecpa|Temporal=Yes|Analysed=Yes
8	qujmonaoaltiaia	nahualtia	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Reflexive[iobj]=Yes|Subcat=Ditr|Tense=Past|VerbForm=Fin	1	_	_	Folio=14r|Paragraph=3|Line=2|Norm=quimonahualtiaya|Analysed=Yes
9	in	in	DET|SCONJ	_	_	10	_	_	Folio=14r|Paragraph=3|Line=2|Norm=in
10	Coiutl	coyotl	NOUN	_	State=Abs	8	_	_	Orig=Co¶·iutl|Folio=14r,14r|Paragraph=3,3|Line=2,3|Norm=Coyotl|Gloss=coyote|Analysed=Yes
11	:	:	PUNCT	_		17	punct	_	Folio=14r|Paragraph=3|Line=3|Norm=:|Analysed=Yes
12	auh	auh	CCONJ	_	_	17	_	_	Folio=14r|Paragraph=3|Line=3|Norm=auh|Gloss=y
13	injc	inic	SCONJ	_		17	_	_	Folio=14r|Paragraph=3|Line=3|Norm=inic|Analysed=Yes
14	coiutl	coyotl	NOUN	_	State=Abs	17	_	_	Folio=14r|Paragraph=3|Line=3|Norm=coyotl|Gloss=coyote|Analysed=Yes
15	,	,	PUNCT	_		17	punct	_	Folio=14r|Paragraph=3|Line=3|Norm=,|Analysed=Yes
16	mjiecpa	miecpa	ADV	_		17	_	_	Orig=mj·iecpa¶|Folio=14r,14r|Paragraph=3,3|Line=3,3|Norm=miecpa|Temporal=Yes|Analysed=Yes
17	teiaiacatzacujlia	yahyacatzacuilia	VERB	_	Animacy[iobj]=Anim|Number[subj]=Plur,Sing|Person[iobj]=3|Person[subj]=3|Subcat=Ditr|Tense=Pres|VerbForm=Fin	8	_	_	Folio=14r|Paragraph=3|Line=4|Norm=teyayacatzacuilia|Analysed=Yes
18	in	in	DET|SCONJ	_	_	19	_	_	Folio=14r|Paragraph=3|Line=4|Norm=in
19	vtlica	ohtlica	ADV	_		17	_	_	Folio=14r|Paragraph=3|Line=4|Norm=ohtlica|Gloss=en.el.camino|Root=ohtli|Analysed=Yes
20	.	.	PUNCT	_		1	punct	_	Folio=14r|Paragraph=3|Line=4|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
Questions:
* Status of *ca*, it looks like the `PART`, but check...
* What is *inic*, it seems to be functioning as an adposition, unless *coyotl* is a clause.
* What's the deal with *mo* in *quimonahualtiaya*? Is it honorific here? 
* How about the *in* that goes with *in ohtlica*?

**Comment by ftyers** [comment]
López Austin:
> Decían que Tezcatlipoca muchas veces se disfrazaba de coyote. Y este coyote muchas veces atajaba los pasos a la gente en el camino.

Suggestions:
* *nahualtia*:
  - Subject: *in Tezcatlipoca*, *Ø-*
  - Object: Reflexive, *mo-*
  - Second (indirect) object: *in coyotl*, *qui-*
* Maybe *auh inic coyotl, miecpa teyayacatzacuilia in ohtlica* is something like "and how [this] coyote, often blocked people on the road".
  - Subject: *coyotl*, *Ø-*
  - Object: Animate, *te-*
  - Oblique:  *in ohtlica* (this would mean that the *in* with *ohtlica* is a determiner and that is not clausal).

That would lead to something like:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/280df22d-a069-4027-9da8-a319cbc3710a)

Thoughts?

**Comment by Lguyogiro** [comment]
re _inic coyotl_, I think this should be _inin coyotl_

**Comment by Lguyogiro** [comment]
> nahualtia:
> 
>     Subject: in Tezcatlipoca, Ø-
>     Object: Reflexive, mo-
>     Second (indirect) object: in coyotl, qui-

yes this looks correct. though I'm not sure about "tezcatlipuca" being nsubj in the tree. i.e. isn't _ca in tezcatlipuca:_ a clause?

**Comment by ftyers** [comment]
> re _inic coyotl_, I think this should be _inin coyotl_

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/ae37f6ab-e543-401b-9118-38a1dfa62b19)

Yep. I'll fix this in the original.

**Comment by ftyers** [comment]
> > nahualtia:
> > ```
> > Subject: in Tezcatlipoca, Ø-
> > Object: Reflexive, mo-
> > Second (indirect) object: in coyotl, qui-
> > ```
> 
> yes this looks correct. though I'm not sure about "tezcatlipuca" being nsubj in the tree. i.e. isn't _ca in tezcatlipuca:_ a clause? otherwise this syntax is kind of marked.

Is your proposal:

> They said: [indeed it was Tezcatlipoca], [he disguised himself as a coyote], [and this coyote obstructed people in the road].

I can find very few examples of *ca in ...* in Andrews or Launey, but one (p.527) is:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/1332fbbb-09c1-40cd-ab6c-39e249be0a3a)

Launey (p.312) has:

> Mìtoa: cuix īxquich quitta in huitziltzin? Ca in huitziltzin cencâ pitzactōn in ītēn "Se dice: ¿el colibrí (huitzilin) puede ver tanto? Pues el colibrí tiene un pico ("labios") muy delgado".

So we could do:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/e8e35af7-eab6-4efd-ba5a-09621793a3cf)

**Comment by Lguyogiro** [comment]
yeah that looks good to me.

**Comment by ftyers** [comment]
Merged in b593e9c.
